libvast/vast/config.hpp.in

```diff
@@ -18,6 +18,16 @@
 #cmakedefine01 VAST_ENABLE_UBSAN
 #cmakedefine01 VAST_ENABLE_UNIT_TESTS
 
+namespace vast::version {
+
+/// Contains the full version string.
+extern const char* version;
+
+/// Contains a hash of VAST's build tree.
+extern const char* build_tree_hash;
+
+} // namespace vast::version
+
 #define VAST_VERSION "@VAST_VERSION_TAG@"
 #define VAST_BUILD_TREE_HASH "@VAST_BUILD_TREE_HASH@"
```

Comment:
I think these two lines still have to be removed for the change to have the desired effect.

Reply:
Yes, absolutely.